test/desktop/features/scripts/build_maze_runner.sh

```diff
@@ -5,7 +5,27 @@ if [ -z "$UNITY_VERSION" ]; then
   exit 1
 fi
 
-UNITY_PATH="/Applications/Unity/Hub/Editor/$UNITY_VERSION/Unity.app/Contents/MacOS"
+
+#!/usr/bin/env bash
+
+if [ "$(uname)" == "Darwin" ]; then
+   PLATFORM="MacOS"    
+elif [ "$(expr substr $(uname -s) 1 10)" == "MINGW64_NT" ]; then
+    PLATFORM="Win64"  
+fi
+
+if [ "$PLATFORM" == "MacOS" ]; then
+
+  echo "MacOS Detected"
+  UNITY_PATH="/Applications/Unity/Hub/Editor/$UNITY_VERSION/Unity.app/Contents/MacOS/Unity"
+
+else
+
+  set -m
+  echo "Windows64 Detected"
+  UNITY_PATH="/c/Program Files/Unity/Hub/Editor/$UNITY_VERSION/Editor/Unity.exe" 
+
+fi
```

Comment:
Why not combine this into the condition blocks above (lines 11-15), since its essentially the same check?

Reply:
done